File: Day-05/268_MissingNumber.java
import java.util.*;

class  Future{

    static int MissingNumber(int arr[]){

        int num = arr.length * (arr.length +1) /2;

        for(int i=0 ;i<arr.length ; i++){

            num= num - arr[i];
        }
    
        return num;
    }   

    public static void main(String[] args) throws Exception{
        
        int test_case_1[] = new int[]{3,0,1};
		int test_case_2[] = new int[]{0,1};
        int test_case_3[] = new int[]{9,6,4,2,3,5,7,0,1};

        System.out.println("--- test case 1 ---");
        System.out.println("Input: nums = "+Arrays.toString(test_case_1)+"\nOutput: "+MissingNumber(test_case_1));
        System.out.println("--- test case 2 ---");
        System.out.println("Input: nums = "+Arrays.toString(test_case_2)+"\nOutput: "+(MissingNumber(test_case_2)));
        System.out.println("--- test case 3 ---");
        System.out.println("Input: nums = "+Arrays.toString(test_case_3)+"\nOutput: "+(MissingNumber(test_case_3)));
    }
            
}

File: Day-05/268_MIssingNumber.py
def MissingNumber(test_case):
    num = len(test_case) * (len(test_case) +1 )/2

    for i in range(len(test_case)-1,-1,-1):
        num=num-test_case[i]

    return num
        
test_case_1 = [1,2,3]
test_case_2 = [0,1]
test_case_3 = [9,6,4,2,3,5,7,0,1];

print("--- test case 1 ---");
print(f"Input: nums = {test_case_1}\nOutput: {MissingNumber(test_case_1)}")
print("--- test case 2 ---")
print(f"Input: nums = {test_case_2}\nOutput: {MissingNumber(test_case_2)}")
print("--- test case 3 ---")
print(f"Input: nums = {test_case_3}\nOutput: {MissingNumber(test_case_3)}")